src/frequenz/client/reporting/_client.py

```diff
@@ -64,10 +64,13 @@
 class ReportingApiClient(BaseApiClient[ReportingStub]):
     """A client for the Reporting service."""
 
+    # pylint: disable-next=too-many-arguments
     def __init__(
         self,
         server_url: str,
+        *,
         key: str | None = None,
```

Comment:
Sorry for the delay and not seeing this before, but it would also be nice to have consistent client interfaces, so I would also rename this to `auth_key`:
```suggestion
        auth_key: str | None = None,
```
:grimacing: